pkg/cmd/dictionary/entries/clear/clear.go

```diff
@@ -0,0 +1,142 @@
+package clear
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/spf13/cobra"
+
+	"github.com/algolia/cli/pkg/cmdutil"
+	"github.com/algolia/cli/pkg/config"
+	"github.com/algolia/cli/pkg/iostreams"
+	"github.com/algolia/cli/pkg/prompt"
+	"github.com/algolia/cli/pkg/utils"
+)
+
+type ClearOptions struct {
+	Config config.IConfig
+	IO     *iostreams.IOStreams
+
+	SearchClient func() (*search.Client, error)
+
+	Dictionnaries []search.DictionaryName
+	All           bool
+	DoConfirm     bool
+}
+
+var (
```

Comment:
Go doesn't support constants for slice/arrays. Since I use this list in multiple places, I would rather have it a the top level (you cannot do your snippet at the top level, it would need to be scoped).